waterdata/templates/macros/components.html

```diff
@@ -1,6 +1,6 @@
 {% macro TimeseriesComponent(site_no) -%}
     <div class="wdfn-component" data-component="hydrograph" data-siteno="{{ site_no }}">
-        <div class="compare-container" data-siteno="{{ site_no }}">
+        <div class="compare-container" style="display: none;" data-siteno="{{ site_no }}">
```

Comment:
I wonder if using the "hidden" attribute would work? That's the recommendation from the designsystem.digital.gov website.